### ISSUE TITLE
Controller range question

### ISSUE THREAD

**Issue by loganmc10** [raphnet/mupen64plus-input-raphnetraw#4]
This has nothing to do with this plugin, apologies.

@raphnet I figured you probably have a lot of knowledge on this subject, and potentially the tools to test. I am looking into this issue: https://github.com/mupen64plus/mupen64plus-user-issues/issues/695

In Goemon's Great Adventure, he seems to be able to jump farther when using the dpad compared to the analog stick.

My question:

I know that the max value for the X and Y axis is +/- 80. What about the corners? If I have the analog stick extended fully to the top right corner, should I expect the X/Y values to be at their respective maximums? Or should they be lower?

Basically: does the analog stick live inside a square, or a circle?

**Comment by loganmc10** [comment]
Somebody did a test using an N64 controller so this question is answered, thanks
